test/phone-number-generator.test.ts

```diff
@@ -3,9 +3,21 @@ import { CountryNames } from "../src/config";
 import { countryPhoneDataArray } from "../src/countryPhoneData";
 import { generatePhoneNumber } from "../src/index";
 import {isPhoneNumber} from 'class-validator';
+import {invalidNumbers} from '../src/utils'
 
 describe("generatePhoneNumber", () => {
-  it.each(Array.from(Array(100000)))(
+  it.each(Array.from(Array(10000)))(
```

Comment:
I would change this test to check that each of this phone numbers in the array are failing to pass the new validation function